theSrc/internal_www/data/displayr_regression/set9/colorscale_numeric_smallmultiples_state/after_drag.json

```diff
@@ -0,0 +1,58 @@
+{
+    "X": [
+      1,
+      0,
+      8,
+      0,
+      0,
+      2,
+      19,
+      0,
+      15,
+      1,
+      14,
+      7,
+      6
+    ],
+    "Y": [
+      1,
+      0,
+      4,
+      0,
+      0,
+      0,
+      5,
+      0,
+      8,
+      1,
+      5,
+      4,
+      3
+    ],
+    "label": [
+      "63",
+      "126",
+      "189",
+      "1008",
+      "1071",
+      "1134",
+      "1197",
+      "2016",
+      "2079",
+      "2142",
+      "3024",
+      "3087",
+      "3150"
+    ],
+    "hiddenlabel.pts": [],
+    "plotlyAnnotations": [
+      {
+        "index": 0,
+        "text": "63",
+        "visible": true,
+        "xoffset": 92,
```

Comment:
Is this in pixels? Does it work with zoom?

Reply:
It is in pixels. Let me see if I can convert it to chart units

Reply:
I can change it to chart units, but then there is no way to make it sit offset away from the marker as its default position? Maybe we just leave it in pixels for now? The small multiple labels already have some differences to the rhtmlLabeledScatter labels.

Reply:
OK, but does zoom work or has it been disabled?

Reply:
Ok, I've changed it to chart units. The starting positions are a bit of a hack, but I guess it was a hack previously anyway

Reply:
So, to clarify. I've changed the labels to store their positions in chart units, so it works fine (which is works in small multiples)